services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.db import transaction
+from django.db.models import QuerySet
+from django.utils.datetime_safe import datetime
+
+from db.models import Ticket, Order, User, MovieSession
+
+
+@transaction.atomic
+def create_order(tickets: list[Ticket], username: str,
+                 date: datetime = None) -> None:
+    user = User.objects.get(username=username)
+    order = Order.objects.create(
+        user=user
+    )
+    if date:
+        order.created_at = date
+        order.save()
+    for ticket in tickets:
+        movie_session = MovieSession.objects.get(
+            id=ticket["movie_session"])
```

Comment:
Consider adding error handling for cases where a `MovieSession` might not be found. Use `MovieSession.objects.get(id=ticket['movie_session'])` within a try-except block to catch `MovieSession.DoesNotExist` exceptions.